src/domain/targeted-messaging/entities/create-outreach.dto.entity.ts

```diff
@@ -15,6 +15,7 @@ export const CreateOutreachDtoSchema = z.object({
   sourceFile: z.string().nullish().default(null),
   sourceFileProcessedDate: z.coerce.date().nullish().default(null),
   sourceFileChecksum: z.string().nullish().default(null),
+  targetAll: z.boolean(),
```

Comment:
Can be done after this PR, but I noticed we have no test coverage for this schema.